draft-ietf-quic-transport.md

```diff
@@ -4721,7 +4717,12 @@ packets to the connection ID provided by the server.  A client MUST NOT change
 the cryptographic handshake message it sends in response to receiving a Retry.
 
 A client MUST NOT reset the packet number for any packet number space after
-processing a Retry packet; {{packet-0rtt}} contains more information on this.
+processing a Retry packet. This applies to 0-RTT packets in particular as those
+contain confidential information that is most likely to need transmission.  The
+keys used to protect packets will not change as a result of responding to a
+Retry packet.  Sending packets with the same packet number is likely to
+compromise the packet protection for those packets because the same key and
+nonce could be used to protect different content.
```

Comment:
```suggestion
processing a Retry packet. In particular, 0-RTT packets contain confidential
information that will most likely be retransmitted on receiving a Retry packet.
The keys used to protect these new 0-RTT packets will not change as a result of
responding to a Retry packet. However, the data sent in these packets could be
different than what was sent earlier. Sending these new packets with the same
packet number is likely to compromise the packet protection for those packets
because the same key and nonce could be used to protect different content.
```